doc/changes/latest.inc

```diff
@@ -120,6 +120,8 @@ Bugs
 
 - When creating `~mne.Epochs`, we now ensure that ``reject_tmin`` and ``reject_tmax`` cannot fall outside of the epochs' time interval anymore (:gh:`8821` by `Richard Höchenberger`_)
 
+- :func:`read_raw_bti() <mne.io.read_raw_bti>` erroneously treated response channels as respiratory channels (:gh:`8855` by `Richard Höchenberger`_)
```

Comment:
FYI if you can live with it being shortened to `read_raw_bti` (no parens) then you can just do
```suggestion
- `~mne.io.read_raw_bti` erroneously treated response channels as respiratory channels (:gh:`8855` by `Richard Höchenberger`_)
```
And the `:func:` is optional because of how we set our default role in `doc/conf.py`

Reply:
Right, I KNEW that once, but have forgotten again :D thanks for the reminder!

Reply:
Adding the parens automatically is a sphinx configuration variable FWIW.  So if people are using the more convoluted syntax just because they want parens, we could enable auto-parens-addition globally.